src/compiler/php_generator.cc

```diff
@@ -78,10 +78,16 @@ void PrintMethod(const MethodDescriptor* method, Printer* out) {
   out->Print("/**\n");
   out->Print(GetPHPComments(method, " *").c_str());
   if (method->client_streaming()) {
+    if (method->server_streaming()) {
+      vars["return_type_id"] = "\\Grpc\\BidiStreamingCall";
+    } else {
+      vars["return_type_id"] = "\\Grpc\\ClientStreamingCall";
+    }
     out->Print(vars,
                " * @param array $$metadata metadata\n"
                " * @param array $$options call options\n"
-               " * @return \\$output_type_id$\n */\n"
+               " * @return $return_type_id$ with "
+               "response type \\$output_type_id$\n */\n"
```

Comment:
Thanks for doing this. But there is probably no need for ` with response type ...`. You can remove it.

One day, if PHP has generics type perhaps we can add it back properly.

Reply:
comment addressed